zh/deploy-on-alibaba-cloud.md

```diff
@@ -191,6 +191,20 @@ category: how-to
 
     根据实际情况修改 `replicas`、`storageClaims[].resources.requests.storage`、`storageClassName`。
 
+    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 spec 下面的 tidb/pd/tikv/tiflash 的 baseImage 项加上 "-enterprise"
```

Comment:
Suggesting putting this section behind the L230.
We also need to cover the ticdc.

Reply:
```suggestion
    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 `spec.<tidb/pd/tikv/tiflash>.baseImage` 项加上 `-enterprise`:
```

Reply:
> Suggesting putting this section behind the L230.
> We also need to cover the ticdc.

Agreed.